internal/api/api_converters.go

```diff
@@ -265,7 +275,51 @@ func (h *DomainHandler) calculateSuiteStatus(specRuns []*testingDomain.SpecRun)
 		return "failed"
 	}
 	if hasSkipped {
-		return "partial"
+		return "skipped"
```

Comment:
This change is also better done in a separate PR, unless it's somehow related?

Reply:
We figured partial was meaningless - it's skipped!